templates/registration/logged_out.html

```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
```

Comment:
The template correctly extends `base.html`, which is a common practice to maintain consistent layout across different pages.